From c265fea7039300ff2283446ce6e7bc03439e2860 Mon Sep 17 00:00:00 2001
From: maxice8 <thinkabit.ukim@gmail.com>
Date: Wed, 27 Jun 2018 21:55:47 -0300
Subject: [PATCH] jpg.h: add include for sys/types.h and define _GNU_SOURCE.

this allows us to use uint on musl libc systems like Void Linux and
Alpine.
---
--- jpg.h
+++ jpg.h
@@ -1,6 +1,9 @@
 #ifndef _JPG_H
 #define _JPG_H

+#include <sys/types.h>
+
+#define _GNU_SOURCE 1
 typedef struct {
     uint height;
     uint width;
